requirements.txt

```diff
@@ -1,8 +1,10 @@
-aiohttp==2.2.5
-async-timeout==1.2.1
-chardet==3.0.4
--e git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py
--e git+https://github.com/strinking/python-duckduckgo@async#egg=duckduckgo
-multidict==3.1.3
-websockets==3.4
-yarl==0.12.0
+PyYAML>=3.10
+aiohttp>=2.2.0
+async-timeout>=1.2.0
+chardet>=3.0.0
+multidict>=3.1.0
+ratelimit>=1.4.0
+websockets>=3.0
+yarl>=0.12.0
+git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]
```

Comment:
why do we need voice support? is it gonna play join kerrhau in voice and start "quack"-ing ?

Reply:
We don't, I just copied it from my normal discord.py install thing. Removed.